Header header
geometry_msgs/Pose    pose
uint8 TYPE_POLE=0
uint8 TYPE_CORNER=1
uint8 TYPE_WALL=2
uint8 TYPE_LANELINE=3
string  ID
uint8 type
int32 life
float32 mappingConfidence
string mapId
float32 trackingConfidence

